File: src/examples/c64pascal/hello.pas
// ---------------------------------------------------------------------------
// (c) 2026 by Jens Kallup - paule32
// Alle Rechte vorbehalten.
// ---------------------------------------------------------------------------
program HelloC64;

var
  I: Integer;

begin
  ClrScr;
  WriteLn('C64 PASCAL');

  for I := 0 to 25 do
  begin
    Poke($0400 + I, 1 + I);
    Poke($D800 + I, 1);
  end;
end.

File: src/examples/c64c/hello.c
// ---------------------------------------------------------------------------
// (c) 2026 by Jens Kallup - paule32
// Alle Rechte vorbehalten.
// ---------------------------------------------------------------------------
#include <stdbool.h>
#include <stdio.h>
#include <c64.h>

int main(void)
{
    unsigned char i;

    clrscr();
    printf("C64 C\n");

    for (i = 0; i < 26; i++) {
        poke(0x0400 + i, 1 + i);
        poke(0xD800 + i, 1);
    }

    return 0;
}
